src/NHibernate.Validator/Constraints/RangeAttribute.cs

```diff
@@ -94,6 +95,10 @@ public bool IsValid(object value, IConstraintValidatorContext validatorContext)
 
 		public void Apply(Property property)
 		{
+			if (AttributeUtils.AttributeUsedMultipleTimesOnProperty(property, GetType()))
+				return;
+
+
 			IEnumerator ie = property.ColumnIterator.GetEnumerator();
 			ie.MoveNext();
 			var col = (Column)ie.Current;

```

Comment:
`GetEnumerator` yields an `IEnumerator<ISelectable> which is disposable, and not disposed of... This kind of old code should be modernized when met, for using extension methods like `First` instead.